RecoverVm/readme.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+Occasionally Azure IaaS VMs may not start because there is something wrong with the operating system (OS) disk preventing it from booting up correctly.
+In such cases it is a common practice to recover the problem VM by performing the following steps:
+
+- Delete the VM (but keep the disks)
+- Attach the disk(s) to another bootable VM as a Data Disk
+- Run the script https://github.com/sebdau/azpstools/blob/master/FixDisk/TS_RecoveryWorker2.ps1 as an elevated administrator from the recovery VM
+- Detach the disk and recreate the original VM using the recovered operating system disk
+
+> the full details on this recovery process are explained in this blog post:
+> https://blogs.msdn.microsoft.com/mast/2014/11/20/recover-azure-vm-by-attaching-os-disk-to-another-azure-vm/
+
+# Current version supports
+- Azure Service Management (ASM) Cloud Service hosted VM recovery
+- Windows 2008 R2 - 2012 R2
```

Comment:
Is it feasible to take a snapshot of the original disk (corrupt), and if so operate on it. And then let the customer apply the fix on the original data dick and perform the recovery